addons/mail/static/tests/discuss/core/suggestion.test.js

```diff
@@ -112,7 +117,7 @@ test("Sort partner suggestions by recent chats", async () => {
         {
             channel_member_ids: [
                 Command.create({
-                    last_interest_dt: "2023-01-01 00:00:00",
+                    last_interest_dt: "2023-01-01 00:00:20",
```

Comment:
it's probably ok but note that this is changing the test to compare on `last_interest_dt` whereas it previously convered the `id` compare

Reply:
It was unclear `id` coverage is intended, I changed it because it was not clear what was the expected order.